services/src/main/java/org/keycloak/services/util/CookieHelper.java

```diff
@@ -38,6 +39,8 @@
  */
 public class CookieHelper {
 
+    public static final String LEGACY_COOKIE = "_LEGACY";
```

Comment:
Some java doc would be nice, in order to know where this _LEGACY comes from

Reply:
Sorry... Just a copy/paste from recent Keycloak code. I tried it but it didn't solve the issue. I kept it as it makes things a little bit better.